src/routes/lobby/components/LobbyPlayerIndicator.vue

```diff
@@ -126,6 +129,34 @@ export default {
   z-index: 1;
 }
 
+.water-container {
+  top: 0;
+  max-height: 33vh;
+  max-width: calc(35vh / 1.45);
+  overflow: hidden;
+  background-color: transparent;
+  border-radius: 10px;
+  position: relative;
```

Comment:
Rather than relatively positioning the element and setting it's height and width to 'manually' match the container, wouldn't it make more sense to absolutely position this element and give it height and width 100% so it always matches the container? That way if we change the size of the container this would also change in step with it.
```suggestion
  position: absolute;
  top: 0;
  height: 100%;
  width: 100%
  overflow: hidden;
  background-color: transparent;
  border-radius: 10px;
```

Reply:
I forgot to change those back! I was trying something at that moment, but that's why it have a 'top' position, it was originally absolute and with percentage based size, thanks for the correction

Reply:
Resolved in commit bb0774ef 👍🏽